src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java

```diff
@@ -40,7 +45,7 @@
 
   private static final Logger log = LoggerFactory.getLogger(ActivityWorker.class);
 
-  private static final String POLL_THREAD_NAME_PREFIX = "Activity Poller ";
+  private static final String POLL_THREAD_NAME_PREFIX = "Poller taskList=";
```

Comment:
Should say it it is activity or workflow one.

Reply:
When we use it we put the type in the end.
```java
            new PollerOptions.Builder(pollerOptions)
                .setPollThreadNamePrefix(
                    POLL_THREAD_NAME_PREFIX
                        + "\""
                        + taskList
                        + "\", domain=\""
                        + domain
                        + "\", type=\"activity\"")
                .build();
```